.dependabot/config.yml

```diff
@@ -31,3 +31,8 @@ update_configs:
           # on false positives and breaking ai-web, so avoiding
           # dependabot auto-update for them like other deps.
           dependency_name: "axe-core"
+      - match:
+          # All electron builds are going to be manually updated
+          # since we use internal microsoft electron distributable
```

Comment:
I feel like this comment sounds a little ominous for an external user finding it ("why is Microsoft using an internal electron distributable? what are they injecting in there?").

Maybe slightly expand it to clarify "...since our release builds use a non-standard Electron build with distribution-restricted media codecs stripped out"?

Reply:
Updated the comment. Thanks for checking this!